### ISSUE TITLE
Installable via npm

### ISSUE THREAD

**Issue by konklone** [18F/analytics-proxy-nodejs#11]
Publish to `npm`, and indicate a runnable command in the `package.json`, so that you can do `npm install analytics-report` [or something] and then run `report [report-name]` to generate/publish a given report.

**Comment by konklone** [comment]
Note to self:

- [ ] Path to `secret_key.pem` needs to be specified as an environment variable as well

**Comment by konklone** [comment]
This is already done. It just needs to be actually published to `npm`, after #40 is resolved.
